.github/workflows/ci.yaml

```diff
@@ -0,0 +1,128 @@
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: '3.x'
+      - name: Generate Proton AppId list from proton.json
+        run: |
+             import json
+             import sys
+             with open("proton.json") as f:
+                 proton = json.load(f)
+                 try:
+                     default_is_valid = False
+                     epilog = "Proton AppId list:\n"
+                     for k, v in proton.items():
+                         if k == "default":
+                             break
+                         default_mark = ""
+                         if k == proton["default"]:
+                             default_mark += " (Default)"
+                             default_is_valid = True
+                         epilog += "    Proton {:13}: {:>10}{}\n".format(
+                           k, v, default_mark)
+                     print(epilog, end="")
+                 except KeyError:
+                     sys.exit(
+                       "** Default version is undefined **\n> {}".format(proton))
+                 if not default_is_valid:
+                     sys.exit(
+                       "** Default version is invalid **\n> {}".format(proton))
+        shell: python
+
+  download_files:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: '3.x'
+      - name: Test download_files(), including HTTP redirection handling
+        run: |
+             import importlib
+             import os
+             import shutil
+             import sys
+             import tempfile
+             shutil.move("truckersmp-cli", "main.py")
+             sys.path.append(os.getcwd())
+             main = importlib.import_module("main")
+             basedir = "/lhark/truckersmp-cli/raw/f67b887b25747f5601549b8cbe2a7e8f9fa64ce7/"
+             files = [
+               ("Makefile", "af9fa17e5a5d03a703b363b7d9712f5d"),
+               ("LICENSE", "702573e867acc76789f7a8c8c861e8fb"),
+             ]
+             files_to_download = []
+             with tempfile.TemporaryDirectory() as tempdir:
+                 for name, md5 in files:
+                     files_to_download.append(
+                       (os.path.join(basedir, name),
+                        os.path.join(tempdir, name),
+                        md5))
+                 main.download_files("github.com", files_to_download)
+        shell: python
+
+  flake8:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: '3.x'
+      - name: Install flake8, flake8-docstrings, and flake8-import-order
+        run: pip install flake8 flake8-docstrings flake8-import-order
+      - name: Check main script by flake8
+        run: flake8 --max-line-length=90 --show-source --statistics truckersmp-cli
```

Comment:
Changed to meaningful names (I think).

Example:
![get_displayed](https://user-images.githubusercontent.com/10556453/80809682-201d4f80-8bfd-11ea-8d7e-a6a9b60d1210.png)

Reply:
I had this in mind: https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idname

Reply:
Sorry. `jobs.<job_id>.name` is displayed instead of `<job_id>`.

Example:
![job_name](https://user-images.githubusercontent.com/10556453/80854096-4b8d5200-8c70-11ea-931d-2c86943b8ee2.png)

* "Inject program" is the `name:` of yaml
* "Build and ... program" is `jobs.<job_id>.name`